### PR TITLE
Feature1 test

### PR DESCRIPTION
**Pull request by skomerko**
bla bla bla

-
-
-
-

### CODE REVIEW

File: after_pr_opened.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+omegalul
```
Comment (skomrzyk) [inline after_pr_opened.txt:1]: amazing piece of code

Comment (skomerko) [reply after_pr_opened.txt:1]: Noice

**Review by skomerko — COMMENTED** [review]
enmemne

File: after_pr_opened.txt (hunk 1 above)
Comment (skomrzyk) [inline after_pr_opened.txt:1]: ee

File: hek.tet [hunk 2]
```diff
@@ -0,0 +1 @@
+mgtmtgm
```
Comment (skomrzyk) [inline hek.tet:1]: klocek

File: new_feature1.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+new f.txt
```
Comment (skomrzyk) [inline new_feature1.txt:1]: prawie

File: nowy.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+tstrstrs
```
Comment (skomrzyk) [inline nowy.txt:1]: koniec

**Review by skomrzyk — APPROVED** [review]
good job

**Pull request merged**